index.html

```diff
@@ -211,6 +210,27 @@ <h1>About Myself</h1>
   </ul>
 </section>
 </article>
+
 </main>
+
+<footer>
+  <h2>Contact me</h2>
+  <p>If you have an application you are interested in developing, a feature that
+     you need built or a project that needs coding. I’d love to help with it </p>
+  <form action="https://formspree.io/f/mrgjeaba" method="post">
+  
+      <label for="name"></label>
+      <input type="text" placeholder="Yeremias" id="name" maxlength="30" required>
+
+    <label for="mail"></label>
+      <input type="email" id="mail" placeholder="natajayanj14@gmail.com" required>
+  
+      <label for="message"></label>
+      <textarea name="user_msg" id="message" maxlength="500" placeholder="Write your message here" required></textarea>
+      <button type="submit">Get It Touch</button>
+    </form>
```

Comment:
- [ ] I would kindly suggest to follow the design provided in figma: 

Figma  | Yours
------------- | -------------
![image](https://user-images.githubusercontent.com/77038610/150117111-46f1752e-ab6d-4479-a550-a8b3e6b78169.png)  | ![image](https://user-images.githubusercontent.com/77038610/150117009-fe1b5831-4926-41a3-a0e4-5572982c7aaa.png)

more specifically:
- please delete the border from input fields and the button,
- please add paddings inside the input fields,

Reply:
@ShoiraTa 
Good afternoon.

After your feedback I managed to do following things:
- remove border from button & input.
- add padding to the input.

May I submit for second review now?

Reply:
Great job implementing the suggested changes by @ShoiraTa